site/content/3.12/aql/functions/vector.md

```diff
@@ -0,0 +1,191 @@
+---
+title: Vector search functions in AQL
+menuTitle: Vector
+weight: 60
+description: >-
+  The functions for vector search let you quickly find semantically similar
+  documents utilizing indexed vector embeddings
+---
+<small>Introduced in: v3.12.4</small>
```

Comment:
Introduced in 3.12.5?

Reply:
No, 3.12.4 is correct. We didn't add reference docs for that release, however.